README.md

```diff
@@ -2333,6 +2417,91 @@ For more details, see the [PartitionAware section](https://docs.hazelcast.org/do
 
 ### 7.8.2. Near Cache
 
+Map  entries in Hazelcast are partitioned across the cluster members. Hazelcast clients do not have local data at all. Suppose you read the key `k` a number of times from a Hazelcast client or `k` is owned by another member in your cluster. Then each `map.get(k)` will be a remote operation, which creates a lot of network trips. If you have a data structure that is mostly read, then you should consider creating a local Near Cache, so that reads are sped up and less network traffic is created.
+
+These benefits do not come for free, please consider the following trade-offs:
+
+- Members with a Near Cache will have to hold the extra cached data, which increases memory consumption.
+- If invalidation is enabled and entries are updated frequently, then invalidations will be costly.
+- Near Cache breaks the strong consistency guarantees; you might be reading stale data.
+
+Near Cache is highly recommended for data structures that are mostly read.
+
+In a client/server system you must enable the Near Cache separately on the client, without the need to configure it on the server. Please note that Near Cache configuration is specific to the server or client itself: a data structure on a server may not have Near Cache configured while the same data structure on a client may have Near Cache configured. They also can have different Near Cache configurations.
+
+If you are using Near Cache, you should take into account that your hits to the keys in the Near Cache are not reflected as hits to the original keys on the primary members. This has for example an impact on IMap's maximum idle seconds or time-to-live seconds expiration. Therefore, even though there is a hit on a key in Near Cache, your original key on the primary member may expire.
+
+NOTE: Near Cache works only when you access data via `map.get(k)` method. Data returned using a predicate is not stored in the Near Cache.
+
+A Near Cache can have its own `in-memory-format` which is independent of the `in-memory-format` of the data structure.
+
+#### 7.8.2.1 Configuring Near Cache
+
+Hazelcast Map can be configured to work with near cache enabled. You can enable the near cache for a map by adding a near cache configuration for that map. An example configuration for `myMap` is shown below.
```

Comment:
You can enable the Near Cache on a Hazelcast Map by adding its configuration for that map. An example configuration for `myMap` is shown below.